lib/foreman_maintain/repos_manager/el.rb

```diff
@@ -0,0 +1,35 @@
+module ForemanMaintain::ReposManager
```

Comment:
I kinda dislike the name, why not "RepoManager"? or "RepositoryManager"? The plural "repos" sounds weird to me.